vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -44,7 +44,18 @@ def process_collection_item(item, key)
 
     def parse_node(node)
       new_result = parse_base_item(node)
-      # TODO, add more properties such as resources
+
+      node_memory = node.status.capacity.send(:memory) || nil
+      node_memory = human_to_value(node_memory) unless node_memory.nil?
+
+      new_result.merge!(
+        :capacity_num_cores => node.status.capacity.send(:cpu) || nil,
+        :capacity_memory    => node_memory,
```

Comment:
@Fryguy do we have a better place where to report the cores number and the memory? (as I type I am looking at aggregate_logical_cpus... which is probably assuming that this data is somewhere else...)